data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageProcessor.java

```diff
@@ -14,21 +14,43 @@
 
 package com.wipro.ats.bdre.lineage;
 
+import com.wipro.ats.bdre.md.dao.LineageQueryDAO;
+import com.wipro.ats.bdre.md.dao.jpa.LineageQuery;
+import com.wipro.ats.bdre.md.dao.jpa.LineageQueryType;
 import org.apache.hadoop.hive.ql.parse.ParseException;
 import org.apache.hadoop.hive.ql.parse.SemanticException;
 import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.AutowireCapableBeanFactory;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
 
 import java.io.IOException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.io.IOException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AUselessImportCheck)